### PR TITLE
Mobile search page - additional features

### PR DESCRIPTION
**Pull request by bxsmith007**
I forgot to let the group know that I had one more thing to add before merging, so that is my fault. This should hopefully be a quick approval and addition from the changes made in the first Mobile search page PR.

### CODE REVIEW

**Comment by bxsmith007** [comment]
Screen Shot 2020-02-11 at 12 08 13 PM

This is what it looks like on mobile. Do you think anything should change in the smaller section, like where "Show" and "Sort by" are?

**Comment by njd9419** [comment]
@bxsmith007 I think the "Show" and "Sort By" are fine where the are, but there should probably be just a bit of whitespace between them so the boxes aren't touching like they are in that screenshot.

**Comment by njd9419** [comment]
Also, aren't all of these commits the ones you already merged in #76 ?

File: client/src/components/Search/Search.js [hunk 1]
```diff
@@ -12,11 +12,6 @@ import * as moment from 'moment';
  */
 export default class Search extends React.Component {
 
-    handlePageChange(page) {
-        const { onPageChange } = this.props;
-        onPageChange(page);
-    }
-
```
Comment (benvogler) [inline client/src/components/Search/Search.js]: Why was this deleted? looks like it's still being called in the `Pagination` below

File: client/src/components/Search/Search.scss [hunk 2]
```diff
@@ -3,68 +3,104 @@
...
+  }
+
+  @include media-breakpoint-down(lg) {
+    width: 100%;
+    flex: 0 0 100%;
+    height: 70%;
+  }
+
+  @include media-breakpoint-up(lg) {
+    width: 40rem;
```
Comment (benvogler) [inline client/src/components/Search/Search.scss]: Why is the width set to `40rem`? This is very unusual
